rts/motoko-rts/src/text_iter.rs

```diff
@@ -50,6 +48,10 @@ const ITER_BLOB_IDX: u32 = 0;
 const ITER_POS_IDX: u32 = 1;
 const ITER_TODO_IDX: u32 = 2;
 
+// Use non-pointer sentinel value as `null` to allow simpler visitor logic.
+// Anlogous to the design of `continuation_table` and `persistence`.
+const NO_OBJECT: Value = Value::from_scalar(0);
```

Comment:
Will we need to tag this scalar for graph copy?

Reply:
I guess it just the representation of Motoko 'false', even when tagged.

Reply:
Yes, I agree, it would be better to explicitly tag this (like the other occurrences) or use an explicit `false` value.

Reply:
See above. This seems safe: https://github.com/dfinity/motoko/pull/4562